synapse/lib/lmdblayer.py

```diff
@@ -94,6 +100,29 @@ class LmdbLayer(s_layer.Layer):
             'range': self._rowsByRange,
         }
 
+    async def setNodeData(self, buid, name, item):
+        # TODO more str->utf8 caching
```

Comment:
@jnwatson's experiments with caching the string encoding showed a poor performance when doing that, compared to just encoding them on the fly.